assets.src/src/redmine-tracky/controllers/issue-completion-controller.ts

```diff
@@ -4,7 +4,7 @@ import ListController from '@controllers/list-controller';
 
 declare function observeAutocompleteField(id: string,
     selector: Function,
-    { select: Function }: any
+    options: { select: Function },
```

Comment:
You can even add more typing :tada: but not necessary imo
```suggestion
    options: { select: (event: Event, item: { item: CompletionResult }) => boolean },
```